doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -228,6 +228,41 @@ information: each universe is either invariant, covariant or
 irrelevant (there are no contravariant subtypings in Coq),
 respectively represented by the symbols `=`, `+` and `*`.
 
+The following example demonstrates this behavior:
+
+.. coqtop:: in
+
+    Set Printing Universes.
+    Set Universe Polymorphism.
+    Set Polymorphic Inductive Cumulativity.
+
+    Inductive Invariant@{=u} : Type@{u}.
```

Comment:
No short explanatory comments here?

Also, if the output is lengthy, you may want to move the example later in the doc and put a hyperlink to it wherever it's needed.